spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/batch/AbstractDaoTests.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.server.batch;
+
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.ext.ScriptUtils;
+import org.testcontainers.jdbc.JdbcDatabaseDelegate;
+
+import javax.sql.DataSource;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AbstractDaoTests {
+
+	protected DataSource createDataSourceForContainer(JdbcDatabaseContainer dbContainer) {
+		DriverManagerDataSource dataSource = new DriverManagerDataSource();
+		dataSource.setDriverClassName(dbContainer.getDriverClassName());
+		dataSource.setUrl(dbContainer.getJdbcUrl());
+		dataSource.setUsername(dbContainer.getUsername());
+		dataSource.setPassword(dbContainer.getPassword());
+		return dataSource;
+	}
+	protected void createDataFlowSchema(JdbcDatabaseContainer dbContainer, String schemaName) throws IOException {
+		JdbcDatabaseDelegate containerDelegate = new JdbcDatabaseDelegate(dbContainer, "");
+		ScriptUtils.runInitScript(containerDelegate, "schemas/drop-table-schema-" + schemaName + ".sql");
+
+		getResourceFiles("schemas/" + schemaName).forEach(str -> {
+			if (str.contains("dataflow"))
+				ScriptUtils.runInitScript(containerDelegate, "schemas/" + schemaName + "/" + str);
+		});
+	}
+
+	private List<String> getResourceFiles(String path) throws IOException {
```

Comment:
It feels like there should be some "utils" lib that would already do this for us.